truss-chains/examples/transcribe/transcribe.py

```diff
@@ -129,7 +118,9 @@ async def run_remote(
             logging.debug(f"Starting macro-chunk [{i + 1:03}/{len(macro_chunks):03}].")
             tasks.append(
                 asyncio.ensure_future(
-                    self._macro_chunk_worker.run_remote(media_url, params, macro_chunk)
+                    self._macro_chunk_worker.run_remote(
+                        url, whisper_parmas, params, macro_chunk
```

Comment:
If we have to param kinds now, rename `params` to `chunking_params` (everywhere).